src/modules/Brightness/Module.js

```diff
@@ -1,15 +1,18 @@
 /*
 * Changes the Image Brightness
 */
-
+const info = require('./info.json');
 module.exports = function Brightness(options,UI){
 
 
     var output;
+    const defaults = info.inputs;
+
+
 
     function draw(input,callback,progressObj){
 
-        options.brightness = parseInt(options.brightness) || 100;
+        options.brightness = parseInt(options.brightness) || defaults.brightness.default;
```

Comment:
Oh, this is nice. I have 2 suggestions. First, could we actually just more directly reference `info.inputs.brightness.default` right here?

Reply:
I will make the changes! Thanks!